editor/state.js

```diff
@@ -204,17 +223,56 @@ export function isSidebarOpened( state = false, action ) {
 }
 
 /**
+ * Reducer returning current network request state (whether a request to the WP
+ * REST API is in progress, successful, or failed).
+ *
+ * @param  {string} state  Current state
+ * @param  {Object} action Dispatched action
+ * @return {string}        Updated state
+ */
+export function api( state = {}, action ) {
```

Comment:
Should this be renamed `saving` or something like that. Thinking we may have other unrelated requests in the future.